br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -166,8 +194,16 @@ func (em *ErrorManager) Init(ctx context.Context) error {
 	if em.remainingError.Type.Load() > 0 {
 		sqls = append(sqls, [2]string{"create type error table", createTypeErrorTable})
 	}
-	if em.dupResolution != config.DupeResAlgNone && em.remainingError.Conflict.Load() > 0 {
-		sqls = append(sqls, [2]string{"create conflict error table", createConflictErrorTable})
+	if em.conflictV1Enabled && em.remainingError.Conflict.Load() > 0 {
+		sqls = append(sqls, [2]string{"create conflict error v1 table", createConflictErrorTable})
+	}
+	if em.conflictV2Enabled && em.remainingError.Conflict.Load() > 0 {
```

Comment:
Does V2 have higher priority than V1? so we only need to create one table

Reply:
Currently, V2 and V1 can exist at the same time. After V2 becomes stable, we can remove the C1 implementation.